README.md

```diff
@@ -80,7 +80,15 @@ To use this dbt package, you must have At least either one Fivetran **Netsuite**
 - vendorcategory
 
 ### Database Compatibility
-This package is compatible with either a **BigQuery**, **Snowflake**, **Redshift**, or **PostgreSQL** destination.
+This package is compatible with either a **BigQuery**, **Snowflake**, **Redshift**, **PostgreSQL**, or **Databricks** destination.
```

Comment:
Same comment as the source package (Updating it to reflect a Part 1)

> ## Step 1: Prerequisites
> To use this dbt package, you must have the following:
> 
> - At least one Fivetran Netsuite connector syncing data into your destination.
> - A **BigQuery**, **Snowflake**, **Redshift**, **Databricks**, or **PostgreSQL** destination.

Reply:
ah yeah so this step is formatted differently due to the Netsuite vs Netsuite2 section

Reply:
My bad.